.github/workflows/push.yml

```diff
@@ -571,8 +571,20 @@ jobs:
           aws-access-key-id: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.ECR_AWS_SECRET_ACCESS_KEY }}
           aws-region: eu-west-1
+      
+      - name: Docker build image IDS
+        working-directory: infra
+        run: |
+          ./scripts/build-docker-container.sh $DOCKER_TAG
+          echo "COMMENT<<EOF" >> $GITHUB_ENV
+          printf "Affected services are: ${{needs.prepare.outputs.IMAGES}}\n" >> $GITHUB_ENV
```

Comment:
We should probably clarify for which deployment are the affected services, now that we have two comments. Have to refresh my knowledge on what we do with IDS deployments